Backpack/Button/Classes/BPKButton.m

```diff
@@ -57,6 +57,10 @@ - (instancetype)initWithSize:(BPKButtonSize)size style:(BPKButtonStyle)style {
     return self;
 }
 
+- (void)didMoveToWindow {
```

Comment:
Instead of this hack we should use `setFontFamily`. This should work, as placing this into a different view-hierarchy will cause the `setter` to fire.